Analysis/tasks.py

```diff
@@ -460,7 +463,7 @@ def run(self):
 
 class AnalysisCacheTask(Task, HTCondorWorkflow, law.LocalWorkflow):
     max_runtime = copy_param(HTCondorWorkflow.max_runtime, 30.0)
-    n_cpus = copy_param(HTCondorWorkflow.n_cpus, 2)
+    n_cpus = copy_param(HTCondorWorkflow.n_cpus, 4)
```

Comment:
This is payload-dependent => get `n_cpus` from the payload producer config in `__init__` (with default value 1)